src/app/components/login/login.js

```diff
@@ -79,7 +78,7 @@ app.component('login', {
       if (passphrase) {
         this.input_passphrase = passphrase;
         if (this.$rootScope.logged === undefined) {
-          this.$timeout(this.passConfirmSubmit.bind(this), 10);
+          // this.$timeout(this.passConfirmSubmit.bind(this), 10);
         }
```

Comment:
I believe it's for development purposes. Just uncomment it and it does autologin.